pypsa/clustering/spatial.py

```diff
@@ -57,12 +57,22 @@ def consense(x):
 
 
 def aggregategenerators(
-    network, busmap, with_time=True, carriers=None, custom_strategies=dict()
+    network,
+    busmap,
+    with_time=True,
+    carriers=None,
+    custom_strategies=dict(),
+    aggregate_buses=None,
```

Comment:
I'd vote for naming it `buses`, to align with the `carriers` argument.

Reply:
Good point!